ui/src/app/pages/data/cohort-review/annotation-definition-modals.component.tsx

```diff
@@ -53,7 +59,9 @@ export const AddAnnotationDefinitionModal = withUrlParams()(class extends React.
 
   async create() {
     try {
-      const {onCreate, urlParams: {ns, wsid, cid}} = this.props;
+      const location = this.props.location;
+      const {params: {ns, wsid, cid}} = matchPath(location.pathname, {path: '/workspaces/:ns/:wsid/data/cohorts/:cid/review'});
```

Comment:
The situation that makes this necessary is when a component is rendered under `WorkspaceWrapperComponent` instead of under the leaf route. Yes, I can put the cid as a component prop.